### PR TITLE
Climax algorithm

### PR DESCRIPTION
**Pull request by scheffeltravis**
This is about all I could manage to complete this week unfortunately, ran into quite a few problems with almost everything I did. Luckily, I at least got some form of high volume trading detection by using a simple moving average of the last twenty days, and buying stock on those days with 10% of my available allotted cash.

### CODE REVIEW

File: sma/example.cs [hunk 1]
```diff
@@ -1,92 +0,0 @@
-using QuantConnect.Data;
```
Comment (scheffeltravis) [inline sma/example.cs:1]: Deleted this because this was a duplicate I left over from my previous cleanup PR.

File: climax/yahoo_reader.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+            equity.Time = datetime.strptime(data[0], "%Y-%m-%d")
+            equity.EndTime = equity.Time + timedelta(days=1)
+            equity.Value = value
+            equity["Open"] = float(data[1])
+            equity["High"] = float(data[2])
+            equity["Low"] = float(data[3])
+            equity["Close"] = float(data[4])
+            # equity["AdjClose"] = float(data[5])
+            equity["Volume"] = float(data[6])
```
Comment (scheffeltravis) [inline climax/yahoo_reader.py:40]: Spent an inordinate amount of time trying to figure out why I couldn't seem to gather Volume data, until I realized that we are most likely using a more recent version of Lean CLI than this file was written for (as the current `Security` class, which is extended by `Equity` used above, in Lean doesn't contain `AdjClose` or `VolumeUSD` as properties): https://github.com/QuantConnect/Lean/blob/4e6e0c85c6d365487316ec212c2a6641cda5fff4/Common/Securities/Security.cs

**Review by jhdaguitera — APPROVED** [review]
This is great! I know this climax detection is going to be difficult. Remind me again, using the 2X volume of SMA, was that a strategy you found or from Chester?

**Comment by scheffeltravis** [comment]
@jhdaguitera I had actually researched quite a bit into how others go about detecting high volumes. The easiest is what I did above with an SMA and then just kind of looked at the stocks in question I analyzed against (TQQQ and SOXL), and then adjusted the flagging based on how large of spikes of volume I saw.

This was sort of the first pass we had agreed to previously. I still need to figure out price movements and combine them to get the whole principle in place, but I believe this should work quite well even if I never get to a more sophisticated method.

**Review by silardi — APPROVED** [review]
Looks like a great start, nice job!

**Pull request merged**